packages/connect/e2e/tests/device/cancelCoinjoinAuthorization.test.ts

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable import/no-named-as-default */
+
+import TrezorConnect, { Success, PROTO } from '../../../src';
+
+const { getController, setup, conditionalTest, initTrezorConnect } = global.Trezor;
+
+const controller = getController('applyFlags');
```

Comment:
I am not sure about this applyFlags here. But I am aware of the fact that we use it also in other tests in tests/device.

Reply:
i believe this is some leftover from previous setups (old repo?) this name was used by trezor-user-env controller (before trezor-user-env-link package) to print test name in case of any error/failure.

currently not used anymore (see e2e/common.setup.js: const getController = name => ...) and could be safely removed
there is still few more occurrences which might be confusing (like  authorizeCoinjoin.test, setBusy.test, unlockPath.test)

note 1: looks like eslint isn't working for that file (js?)

note 2: maybe getController should be called inside od `describe` block? beacause each getController sets two event listeners for `TrezorUserEnvLink` this may lead to memory leaks (max MaxListenersExceededWarning) if it's declared on top level? what do you think @mroz22 ?

Reply:
@dahaca pls move it as i suggested:

```
describe('TrezorConnect.cancelCoinjoinAuthorization', () => {
    const controller = getController();
```

and you can squash it looks good to me :+1:

Reply:
done